components/cd-event/cd-event.html

```diff
@@ -0,0 +1,173 @@
+            <span class="cd-date__month">Sep</span>
+            <span class="cd-date__year">2020</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-card__container [ cd-flow ]">
+
+        <div class="cd-card__title">
+          <h2><a href="#" rel="bookmark">Sed dapibus lectus sit amet adipiscing egestas mauris est viverra nibh</a></h2>
+        </div>
+
+        <div class="cd-card__description node__content">
+          <p>Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh (iaculis pretium sem orci aliquet mauris). Nulla auctor eleifend turpis consequat pharetra. Nulla vel magna sit amet dui lobortis commodo vitae vel nulla sit amet ante hendrerit tempus. Donec a congue leo? </p>
+        </div>
+
+      </div>
+
+      <div class="cd-card__footer">
+        <a href="#" class="cd-card__link cd-read-more" aria-hidden="true" focusable="false" width="16" height="16">
+          Learn more
+          <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false">
+            <use xlink:href="#cd-icon--arrow-right"></use>
+          </svg>
+        </a>
+      </div>
+
+    </article>
+
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__date--event">
+        <div class="cd-date">
+          <div class="cd-date__wrapper">
+            <span class="cd-date__day">30</span>
+          </div>
+          <div class="cd-date__wrapper">
+            <span class="cd-date__month">Jul</span>
+            <span class="cd-date__year">2020</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-card__container [ cd-flow ]">
+
+        <div class="cd-card__title">
+          <a href="#" rel="bookmark">Event title</a>
+        </div>
+
+        <div class="cd-card__location">
+          Location
+        </div>
+
+        <div class="cd-card__time">
+          Time: 16:00 to 17:00
+        </div>
+
+        <div class="cd-card__description node__content">
+          <p>Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh (iaculis pretium sem orci aliquet mauris). Nulla auctor eleifend turpis consequat pharetra. Nulla vel magna sit amet dui lobortis commodo vitae vel nulla sit amet ante hendrerit tempus. Donec a congue leo? </p>
+        </div>
+
+      </div>
+
+      <div class="cd-card__footer cd-align--right">
+        <a href="#" class="cd-card__link cd-read-more">
+          Learn more
+          <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
+            <use xlink:href="#cd-icon--arrow-right"></use>
+          </svg>
+        </a>
+      </div>
+
+    </article>
+
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__date--event">
+        <div class="cd-date">
+          <div class="cd-date__wrapper">
+            <span class="cd-date__day">18-20</span>
```

Comment:
this one wraps like so on the demo:

```
18-
20
```

Is that intentional?

Reply:
Yes. This was initially to call attention to the design problem. I wanted to show the limitation of components in relation to their content, and demonstrate how proposed design do not accommodate. I'll change the fixed width to a min-width.